nimbess-cni.go

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) 2019 Red Hat and/or its affiliates.
```

Comment:
just a note that we should include testing for this in the future

Reply:
Agree